piperider_cli/assertion_engine/assertion.py

```diff
@@ -401,9 +507,44 @@ def evaluate_all(self, metrics_result):
                 exceptions.append((assertion, e))
         return results, exceptions
 
-    def configure_plugins_path(self):
+    def validate_assertions(self):
+
+        from piperider_cli.assertion_engine.types import get_assertion
+        results = []
+
+        for assertion in self.assertions:
+            try:
+                assertion_instance = get_assertion(assertion.name)
+                result = assertion_instance.validate(assertion)
+                if result and result.has_errors():
+                    results.append(result)
+            except ValueError:
+                pass
```

Comment:
`get_assertion` will raise ValueError when function_name is not in custom_registry.
We should handle this exception. Otherwise, `diagnose` will show "passed" when users give a wrong assertion name.

For example:
```yaml
users: # table name
  columns:
    age: # column name
      tests:
      - name: assert_column_min_in_whatever # <- this assertion doesn't exist
        assert:
          min: [18, 20]
```

Reply:
oops ! big bug.